pkgs/by-name/in/inja/simple-cmake-test/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv
+, cmake
+, inja
+, lib
+,
+}:
+
+stdenv.mkDerivation {
+  name = "inja-simple-cmake-test";
+  src = lib.cleanSource ./.;
```

Comment:
May be susceptible to #301014? It's better practice to explicitly list out the files needed anyway